README.md

```diff
@@ -1,12 +1 @@
-# Google Software Product Sprint
```

Comment:
Totally optional, but you might consider leaving in a heading, since I think most GitHub repo readme files have one. But it's completely up to you.